tests/signals_connect/screen-output

```diff
@@ -0,0 +1,46 @@
+-----------------------------------------------------------------------------
+-- This is ASPECT, the Advanced Solver for Problems in Earth's ConvecTion.
+--     . version 1.4.pre
+--     . running in DEBUG mode
+--     . running with 1 MPI process
+--     . using Trilinos
+-----------------------------------------------------------------------------
+
+Loading shared library <./libsignals_connect.so>
+
+Connecting signals
+Number of active cells: 16 (on 3 levels)
+Number of degrees of freedom: 268 (162+25+81)
+
+
+*** constraints signal called ***
+
+*** Timestep 0:  t=0 seconds
+
+*** constraints signal called ***
```

Comment:
good catch -- you take your patch reviewing duties seriously, I see :-)

I had wondered about this as well. The first call to `compute_current_constraints()` comes from the place where we compute the initial values -- I think what happens is that we interpolate the boundary values, and then need to make sure that they satisfy the constraints. The second place is where we set up the computations of the first time step.

I concluded that this is as it is supposed to be. (It may be wasteful to compute the constraints for the velocity in the setup of initial conditions because at that point there is no velocity, but I decided to not worry about this.)